setup.py

```diff
@@ -12,10 +12,7 @@
     "Topic :: Utilities",
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
```

Comment:
Consider making "3 :: Only" explicit, like in pytest?

https://github.com/pytest-dev/pytest/blob/c83801eed0dcfcb14f619dcb5892c9bfca78c573/setup.cfg#L19

```suggestion
    "Programming Language :: Python :: Implementation :: PyPy",
    "Programming Language :: Python :: 3 :: Only",
```

Reply:
Thanks, applied.